src/modules/pmc/src/lib/Pmc.cpp

```diff
@@ -2,8 +2,35 @@
 // Licensed under the MIT License.
 
 #include "Pmc.h"
+#include <CommonUtils.h>
+#include <Mmi.h>
+#include <string>
 
 Pmc::Pmc(unsigned int maxPayloadSizeBytes)
     : PmcBase(maxPayloadSizeBytes)
 {
+}
+
+int Pmc::RunCommand(const char* command, bool replaceEol, std::string* textResult, unsigned int timeoutSeconds)
```

Comment:
>Pmc::RunCommand(const char* command, bool replaceEol, std::string* textResult, unsigned int timeoutSeconds) [](http://example.com/codeflow?start=4&length=107)

Skip the replaceEol and timeoutSeconds and assume defaults for this module (such as: true and 0)

Reply:
I did remove the `replaceEol` but as mentioned in a previous comment, for one of our commands we need a timeout of 600 of seconds.

Reply:
Hardcode that 600 (if you really need 10 minutes) to a constant and use that internally, skipping this argument.